doc/development/contribution.rst

```diff
@@ -1,10 +1,4 @@
 Contribution
 ============
 
-We welcome new contributions -- simply fork the FlamingPy repository and make 
-a pull request (PR) containing your contribution. All contributors to FlamingPy 
-will be listed as authors on the releases. Users who contribute significantly 
-to the code (new plugins, functionalities, etc.) may be listed on the arXiv 
-preprints for FlamingPy. See our 
-`release notes and changelogs <https://github.com/XanaduAI/flamingpy/releases>`__
-for more details.
+We welcome new contributions -- simply fork the FlamingPy repository and make a pull request (PR) containing your contribution. All contributors to FlamingPy will be listed as authors on the changelog and releases. Users who contribute significantly to the code (new plugins, functionalities, etc.) may be listed on the arXiv preprints for FlamingPy. See our `CHNAGELOG.md <https://github.com/XanaduAI/flamingpy/.github/CHANGELOG.md>`__ and `release notes <https://github.com/XanaduAI/flamingpy/releases>`__ for details and you can always reach us on FlamingPy `Discussions <https://github.com/XanaduAI/flamingpy/discussions>`__.
```

Comment:
Same here regarding including `CONTRIBUTING.md`.

Reply:
Great, I'll fix this as well.

Reply:
This is now done.